### PR TITLE
Add TAMR_ES_EXTRA_DOCKER_PARAMS

### PR DESCRIPTION
**Pull request by tonyastolfi**
This change is [Reviewable](https://reviewable.io/reviews/datatamer/elasticsearch/4)

### CODE REVIEW

File: scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/TaskInfoFactory.java [hunk 1]
```diff
@@ -98,7 +98,29 @@ public TaskInfoFactory(ClusterState clusterState) {
 
         final Protos.TaskID taskId = Protos.TaskID.newBuilder().setValue(taskId(offer, clock)).build();
         final List<String> args = configuration.esArguments(clusterState, discovery, offer.getSlaveId());
-        final Protos.ContainerInfo containerInfo = getContainer(configuration, taskId, elasticSearchNodeId, offer.getSlaveId());
+        Protos.ContainerInfo containerInfo = getContainer(configuration, taskId, elasticSearchNodeId, offer.getSlaveId());
+
+        final String TAMR_ES_EXTRA_DOCKER_PARAMS = System.getenv("TAMR_ES_EXTRA_DOCKER_PARAMS");
+        final String delimiter = TAMR_ES_EXTRA_DOCKER_PARAMS.substring(0, 1);
```
Comment (data-tamer-vgp) [inline scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/TaskInfoFactory.java]: Does this mean this env var _has_ to be specified?

Comment (data-tamer-vgp) [reply scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/TaskInfoFactory.java]: Seems like maybe worthwhile to allow it to be optional.

**Comment by tonyastolfi** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/TaskInfoFactory.java, line 104 at r1](https://reviewable.io:443/reviews/datatamer/elasticsearch/4#-KeAJp5zXhSHYMlOkMBm-r1-104:-KeAwTcLlkAJXqMipXEz:b-c3tnki) ([raw file](https://github.com/datatamer/elasticsearch/blob/71737b779dc6b7ecefe74a5d2c42ceb127e008ac/scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/TaskInfoFactory.java#L104)):*
<details><summary><i>Previously, data-tamer-vgp wrote…</i></summary><blockquote>

Seems like maybe worthwhile to allow it to be optional.
</blockquote></details>

Good point.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/datatamer/elasticsearch/4)*

**Comment by tonyastolfi** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/TaskInfoFactory.java, line 104 at r1](https://reviewable.io:443/reviews/datatamer/elasticsearch/4#-KeAJp5zXhSHYMlOkMBm-r1-104:-KeAwy3pY0eREzxAdzm2:b-896fix) ([raw file](https://github.com/datatamer/elasticsearch/blob/71737b779dc6b7ecefe74a5d2c42ceb127e008ac/scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/TaskInfoFactory.java#L104)):*
<details><summary><i>Previously, tonyastolfi (Tony Astolfi) wrote…</i></summary><blockquote>

Good point.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/datatamer/elasticsearch/4)*

**Review by data-tamer-vgp — APPROVED** [review]

**Comment by tonyastolfi** [comment]
Reviewed 1 of 2 files at r1.
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/datatamer/elasticsearch/4)*

**Comment by tonyastolfi** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/datatamer/elasticsearch/4)*

**Pull request merged**